adapter/validator/src/main/java/org/mobilitydata/gtfsvalidator/validator/GtfsTypeValidator.java

```diff
@@ -95,39 +99,117 @@ public GtfsTypeValidator(@NotNull GtfsSpecificationProto.CsvSpecProto fileSchema
                     case INPUT_TYPE_UNSPECIFIED: // Text is default and does not require validation
                     case TEXT:
                     case COLOR:
-                    case DATE: //no special validation
+                    case DATE: // no special validation
                         break;
+                    case INTEGER:{
+                        if (!integerValidator.isInRange(
+                                (Integer) value,
+                                Integer.MIN_VALUE,
+                                Integer.MAX_VALUE)) {
+
+                            toReturn.add(new IntegerFieldValueOutOfRangeNotice(
+                                    toValidate.getRawFileInfo().getFilename(),
+                                    columnSpecProto.getName(),
+                                    toValidate.getEntityId(),
+                                    Integer.MIN_VALUE,
+                                    Integer.MAX_VALUE,
+                                    (Integer) value)
+                            );
+                        }
+                        break;
+                    }
                     case FLOAT: {
                         if (!floatValidator.isInRange(
                                 (Float) value,
-                                columnSpecProto.getFloatmin(),
-                                columnSpecProto.getFloatmax())) {
+                                Float.MIN_VALUE,
```

Comment:
Can never be true. By definition a Float is within [Float.MIN_VALUE, Float.MAX_VALUE]